### ISSUE TITLE
Stopped working yesterday.

### ISSUE THREAD

**Issue by joeysanistam** [platformio/platformio-atom-ide-terminal#914]
The package had trouble installing or updating I guess, because at some point it stopped working and it was giving an error. Can't reinstall or fix the package.

**Comment by sohagINFINITY** [comment]
same issue

**Comment by bossjimmark** [comment]
This seems to not only impact platformio, but it causes any other packages to fail during installation as well.  Platformio was installed, then as of this morning (10/16/2020), Platformio stopped working.  I uninstalled and tried to reinstall to no avail. I tried installing a different terminal application and installation would fail.  I uninstalled and reinstalled Atom and also that didn't work.  
I am wondering if this is an Atom issue.

Atom, as a code editor does still work.
